exporter/awsxrayexporter/internal/translator/cause.go

```diff
@@ -486,6 +488,54 @@ func fillDotnetStacktrace(stacktrace string, exceptions []awsxray.Exception) []a
 	return exceptions
 }
 
+func fillGoStacktrace(stacktrace string, exceptions []awsxray.Exception) []awsxray.Exception {
+	var line string
+	var label string
+	var path string
+	var lineNumber int
+
+	r := textproto.NewReader(bufio.NewReader(strings.NewReader(stacktrace)))
+
+	// Skip first line containing top level exception / message
+	_, _ = r.ReadLine()
+	exception := &exceptions[0]
+	line, err := r.ReadLine()
+	if err != nil {
+		return exceptions
+	}
+
+	exception.Stack = make([]awsxray.StackFrame, 0)
+	for {
+		if strings.Contains(line, "goroutine") {
+			line, _ = r.ReadLine()
+		}
+
+		label = line
+		line, _ = r.ReadLine()
+
+		if strings.Contains(line, "\t") {
+			s := strings.Split(line, ":")
+			path = strings.Trim(s[0], "\t")
```

Comment:
Length check the split array ( we want to make sure this codepath can't panic no matter what stacktrace is passed in in case format changes)